### ISSUE TITLE
Need to migrate to Drive API before April 20, 2015

### ISSUE THREAD

**Issue by devotis** [samcday/node-google-spreadsheets#22]
Google recently communicated that Documents List API will be discontinued on April 20, 2015. This change means that service calls for this API will no longer be supported, and any Google Apps features that are implemented using this API will no longer function.

One of the URL's that will be discontinued is https://spreadsheets.google.com/feeds/ which is used by this module. It's suggested to migrate to the Drive API.

To summarize, node-google-spreadsheets will no longer function as of April 20, 2015 if no action is taken.

**Comment by devotis** [comment]
Update on the matter. You don't need to migrate. Read here:
https://github.com/jpillora/node-edit-google-spreadsheet/issues/57

**Comment by samcday** [comment]
Thanks for investigating this @devotis
